XenModel/Actions/Updates/DownloadUpdatesXmlAction.cs

```diff
@@ -330,12 +332,19 @@ protected virtual XmlDocument FetchCheckForUpdatesXml()
             }
             else
             {
+                
                 var proxy = XenAdminConfigManager.Provider.GetProxyFromSettings(Connection, false);
 
                 using (var webClient = new WebClient())
                 {
                     webClient.Proxy = proxy;
                     webClient.Headers.Add("User-Agent", _userAgent);
+                    if (!string.IsNullOrEmpty(authToken))
+                    {
+                        NameValueCollection myQueryStringCollection = new NameValueCollection();
+                        myQueryStringCollection.Add("burritoToken", authToken);

```

Comment:
This should need changing too - hence better use the constant from Registry.cs

Reply:
Realised i missed that one seconds before this comment!